api/envoy/api/v2/core/protocol.proto

```diff
@@ -117,6 +117,10 @@ message Http2ProtocolOptions {
   // `Maximum concurrent streams <https://httpwg.org/specs/rfc7540.html#rfc.section.5.1.2>`_
   // allowed for peer on one HTTP/2 connection. Valid values range from 1 to 2147483647 (2^31 - 1)
   // and defaults to 2147483647.
+  //
+  // For upstream connections, this also limits how many streams Envoy will initiate concurrently
```

Comment:
Can you add a version history note about this change?

Reply:
As in, "Beginning with version 1.14: For upstream connections, this also......."?